index.src.html

```diff
@@ -816,27 +831,29 @@ <h3 id=interface-reporting-observer>Interface {{ReportingObserver}}</h3>
   
   1. Set the <a>context object</a>'s <a>options</a> to |options|.
   
-  2. If it is not already present, add the <a>context object</a> to the 
-     list of <a>registered reporting observers</a> for the <a>environment
-     settings object</a> associated with the <a>context object</a>.
+  2. If it is not already registered, <a>register</a> the <a>context object</a>
```

Comment:
The concept "registered" is not, I think, precisely defined.  I.e. you've written a function for "register" but not 'isRegistered".

Reply:
Okay, I will clarify the definition.